homeassistant/components/homekit_controller/sensor.py

```diff
@@ -113,7 +112,7 @@ def icon(self):
     @property
     def unit_of_measurement(self):
         """Return units for the sensor."""
-        return UNIT_LUX
+        return LIGHT_LUX
```

Comment:
This is a breaking change; unit of measurement changed from `lux` to `lx`, which can affect things like InfluxDB.